src/model/Element.ts

```diff
@@ -26,9 +25,9 @@ export class Element extends EndevorEntity implements IElement {
   elmName: string;
   fullElmName: string;
   elmVVLL: string;
-  repository: Repository;
```

Comment:
Thy are you deleting the type annotation everywherer?

Reply:
I guess this is causing the circular dependency?

Reply:
My thinking is that we should change `EndevorEntity` into an interface and then in all places that require `Repository` as an argument or return value, change the type annotation to `EndevorEntity`.

As the old saying goes, we should [program to an intreface, not an implementation](https://stackoverflow.com/questions/2697783/what-does-program-to-interfaces-not-implementations-mean).

Reply:
Ooh, that is even more efficient than what I would try, which was to make Repository into an interface, and everything that it imports as well XD